pyiron/atomistics/structure/atoms.py

```diff
@@ -1118,32 +1118,105 @@ def get_extended_positions(self, width):
         v_repeated = v_repeated[check_dist]
         return v_repeated, indices
 
+    def get_numbers_of_neighbors_in_sphere(
+            self,
+            cutoff_radius=10,
+            num_neighbors=None,
+            tolerance=2,
+            id_list=None,
+            boundary_width_factor=1.2,
+            num_neighbors_estimate_buffer=1.0,
+    ):
+        """
+        Function to compute the maximum number of neighbors in a sphere around each atom.
+        Args:
+            cutoff_radius (float): Upper bound of the distance to which the search must be done
+            num_neighbors (int/None): maximum number of neighbors found
+            tolerance (int): tolerance (round decimal points) used for computing neighbor shells
```

Comment:
Since this function doesn't consider the shells, you can get rid of this argument.

Reply:
Thanks. I will take it out